server/chat/background/task.py

```
@@ -2048,103 +2048,205 @@ def create_background_chat_session(
+            cursor.execute("""
+                SELECT 1 FROM execution_steps
+                WHERE incident_id = %s AND status = 'running'
+                LIMIT 1
+            """, (incident_id,))
+            if cursor.fetchone():
+                return False
+        return True
+    return False
+
+
+def _affected_users(cursor):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_affected_users`

(ANN202)